build.gradle

```diff
@@ -35,14 +35,15 @@ buildscript {
 }
 
 ext.versions = [
-        flink   : "1.14.2",
-        hadoop  : "2.7.7",
-        junit4  : "4.11",
-        junit5  : "5.4.+",
-        mockito : "2.0.+",
-        mockito3: "3.+",
-        slf4j   : "1.7.0",
-        vavr    : "0.9.2",
+        flink    : "1.14.2",
+        hadoop   : "2.7.7",
+        junit4   : "4.11",
+        junit5   : "5.4.+",
+        mockito  : "2.0.+",
+        mockito3 : "3.+",
+        spectator: "1.3.+",
```

Comment:
We should figure out how to replace spectator-api with micrometer eventually https://micrometer.io/docs/concepts#_purpose

Reply:
insight team's doc is actually suggesting not to use micrometer but stick with spectator. I feel like there is already a good base of mantis metrics abstraction and we just need to fully remove all direct spectator refs there.